staff/PacoGD/css-layout-simple-adaptive/styles/main.css

```diff
@@ -0,0 +1,2 @@
+@import "desktop.css" screen and (min-width: 768px);
+@import "mobile.css" screen and (max-width: 768px);
```

Comment:
Some of the CSS properties can be listed in the "main.css" document to reduce your code size

Reply:
Okey

Reply:
done